src/lib.rs

```diff
@@ -158,7 +158,14 @@ pub fn compile(
         if contract.instantiable {
             let code = contract.emit(&ns, &opts);
 
-            let (abistr, _) = abi::generate_abi(contract_no, &ns, &code, false);
+            let (abistr, _) = abi::generate_abi(
+                contract_no,
+                &ns,
+                &code,
+                false,
+                &vec!["unknown".to_string()],
```

Comment:
Could use the default values here. Also it'd be nice if this would work in the library version too.